src/logic/safe/store/actions/createTransaction.ts

```diff
@@ -1,9 +1,8 @@
 import { Operation } from '@gnosis.pm/safe-react-gateway-sdk'
-import { push } from 'connected-react-router'
 import { generatePath } from 'react-router-dom'
 import { AnyAction } from 'redux'
 import { ThunkAction } from 'redux-thunk'
-
+import { history } from '../../../../routes/routes'
```

Comment:
We try to follow a full path convention, not using relative paths so it's easier to find files.

Usually your code editor will autocomplete with relative paths :wink:

Reply:
Thanks - I will update this soon.

I've added `"javascript.preferences.importModuleSpecifier": "non-relative"` to VSCode's settings.json to use absolute paths instead. Maybe this is of interest to others on the team @gnosis/safe-web.